### PR TITLE
Fix SRI values for jquery

### PR DESCRIPTION
**Pull request by natemcmaster**
Resolves https://github.com/aspnet/templating/issues/520

### CODE REVIEW

**Comment by natemcmaster** [comment]
@kichalla @mkArtakMSFT can someone take a look at this quickly to verify it?

**Comment by kichalla** [comment]
I will send out a PR for IdentityUI fix

File: test/Templates.Test/CdnScriptTagTests.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            var packages = Directory.GetFiles(artifactsDir, "*.nupkg");
+
+            var scriptTags = new List<ScriptTag>();
+            foreach (var packagePath in packages)
+            {
+                scriptTags.AddRange(GetScriptTags(packagePath));
+            }
+
+            Assert.NotEmpty(scriptTags);
+            var shasum = new Dictionary<string, string>();
```
Comment (kichalla) [inline test/Templates.Test/CdnScriptTagTests.cs]: case-insensitive dictionary here

**Comment by mkArtakMSFT** [comment]
Are there any other files in templates, which will need to be updated? Can you please double check that, please.

**Pull request merged**